keps/sig-auth/20190607-certificates-api.md

```diff
@@ -101,15 +100,22 @@ committing it to storage so that they can be used later during CSR approval. The
 information contained in the spec is immutable after the request is created.
 
 An approver updates approval status of the CertificateSigningRequest via the
-CertificateSigningRequestStatus. The approval condition can only be updated via
-the `/approval` subresource allowing approval permission to be authorized
+CertificateSigningRequestStatus. `Approved`/`Denied` conditions can only be set via
+the `/approval` subresource, allowing approval permission to be authorized
 independently of other operations on the CertificateSigningRequest.
+`Approved`/`Denied` conditions are mutually exclusive, cannot be removed,
+cannot have a `False` or `Unknown` status, and are treated as `True` if present.
 
-Contingent on approval, a signer posts a signed certificate to the status. The
+Once an `Approved` condition is present, a signer posts a signed certificate to the status. The
 certificate field of the status can only be updated via the `/status`
 subresource allowing signing permission to be authorized independently of other
 operations on the CertificateSigningRequest.
 
+Terminal failures can be indicated with a `Failed` condition added by a controller,
+such as a signer indicating a refusal to sign the CSR.
```

Comment:
> such as a signer indicating a refusal to sign the CSR.

Should we provide some clear guidance as to which actor we expect to 'own' this condition/set it? `such as` implies it *may* be some other actor, and this presumably isn't true unless signer controllers are built to check for this condition's status *before* deciding whether Deny it once again itself

Reply:
clarified this is for signers